cache/lru_cache.cc

```diff
@@ -708,10 +771,17 @@ LRUCache::LRUCache(size_t capacity, int num_shard_bits,
   shards_ = reinterpret_cast<LRUCacheShard*>(
       port::cacheline_aligned_alloc(sizeof(LRUCacheShard) * num_shards_));
   size_t per_shard = (capacity + (num_shards_ - 1)) / num_shards_;
+  bool use_compressed_secondary_cache{false};
+  if (secondary_cache && std::strcmp(secondary_cache->Name(),
+                                     kCompressedSecondaryCacheName) == 0) {
+    use_compressed_secondary_cache = true;
```

Comment:
Instead of doing it, consider to add a function in SecondaryCache, something like SecondaryCache::HasAdmissionControl() and deal with it in a more general way.

Reply:
Or perhaps something like SecondaryCache::SupportForceErase().

Reply:
Yeah. It's better to update SecondaryCache.